Mention the required use of node prefixes (#669)
strapi/documentation: docs/developer-docs/latest/developer-resources/cli/CLI.md

# Command Line Interface (CLI)

Strapi comes with a full featured Command Line Interface (CLI) which lets you scaffold and manage your project in seconds. 

::: note

> We recommend to install strapi only locally and never globally. The following documentation does not use any required node prefixes like `npm run` or `yarn`, but you should always prefix `strapi` with what fits best to your project setup, e.g. `npm run strapi new <name>`, `yarn strapi new <name>`

Reviewer: ```suggestion
We recommend installing strapi only locally and never globally. The following documentation does not use any required node prefixes (i.e. `npm run` or `yarn`), but commands should prefix `strapi` with what fits best to the project setup (e.g. `npm run strapi new <name>`, `yarn strapi new <name>`).
```

Reviewer: Suggesting this to be compliant with our [12 rules of technical writing](https://handbook.strapi.io/user-success-manual/12-rules-of-technical-writing)

Author: @pwizla Excellent! Thank you very much!

> We recommend to install strapi only locally and never globally. The following documentation does not use any required node prefixes like `npm run` or `yarn`, but you should always prefix `strapi` with what fits best to your project setup, e.g. `npm run strapi new <name>`, `yarn strapi new <name>`

Reviewer 2: @pwizla what about the usage of `you` here, are we going to let that one slide this time :wink:

Author: @derrickmehaffy The same would be true for "We"... :-)

Reviewer 2: Their rules are pretty strict about wording to remove pronouns entirely. We have much more complex internal technical writing requirements and the exceptions for you or we are most likely counted on 1 hand and extremely rare. I don't think this is one of them :sweat_smile: 

But we will let Piwi confirm

Author: Or: "The Strapi team strongly recommends to install Strapi only locally, which requires prefixing all of the following `strapi` commands with the package manager used for the project setup (e.g. `npm run strapi new <name>`, `yarn strapi new <name>`) or a dedicated node package executor (e.g. `npx strapi new <name>`)."

Reviewer 2: something more like:

> It is recommend to install Strapi locally only, which requires prefixing all of the following `strapi` commands with the package manager used for the project setup (e.g `npm run strapi help` or `yarn strapi help`) or a dedicated node package executor (e.g. `npx strapi help`).

Reviewer: Haha, good catch! Sometimes in my good days I let a few "you"s or "we"s slide in. 😂 
For diversity & inclusion reasons, pronouns should be avoided as much as possible. I'm a bit more liberal in callouts notes or when we want to involve the reader a bit further.

But here, "we" and "you" are not really necessary and Derrick's suggestion is pretty good.
I'll go ahead, merge the PR and update the content myself since we have a deployment scheduled in a few hours from now.
Thank you again @rowild for your attention to detail, we really appreciate it ❤️
